src/tests/system/tests/test_sssctl.py

```diff
@@ -191,30 +191,39 @@ def test_sssctl__reset_cached_timestamps(client: Client, ldap: LDAP):
 @pytest.mark.importance("high")
 @pytest.mark.tools
 @pytest.mark.topology(KnownTopology.Client)
-def test_sssctl__check_typo_option_name(client: Client):
+def test_sssctl__check_typo_option_name_when_sssd_is_stopped(client: Client):
```

Comment:
Good point. However, shouldn't we keep this one test just to assure that SSSD and SSSCTL are independent on each other? That they are not blocking each other?
But if we agree that this is useless I'll remove it.

Reply:
This does not bring any value, I vote for removal of SSSD service manipulation here (The config check still has its value since it manipulates the config file differently than the other tests in this PR).

Reply:
Okay. I removed SSSD manipulation.